src/mixins/setContent.js

```diff
@@ -38,5 +43,38 @@ export default {
 				.run()
 		},
 
+		setInitialYjsState(content, { isRich }) {
+			const html = isRich
+				? markdownit.render(content) + '<p/>'
+				: `<pre>${escapeHtml(content)}</pre>`
+
+			const editor = createEditor({
+				enableRichEditing: isRich,
+			})
+			const json = generateJSON(html, editor.extensionManager.extensions)
+
+			const doc = Node.fromJSON(editor.schema, json)
+			const getBaseDoc = (doc) => {
+				const ydoc = new Doc()
+				// In order to make the initial document state idempotent, we need to reset the clientID
+				// While this is not recommended, we cannot avoid it here as we lack another mechanism
+				// generate the initial state on the server side
```

Comment:
Missing "to" and dot at the end: `to generate the initial state on the server side.`